doc/manuals/user/ngsiv2_implementation_notes.md

```diff
@@ -3,6 +3,7 @@
 * [Forbidden characters](#forbidden-characters)
 * [Custom payload decoding on notifications](#custom-payload-decoding-on-notifications)
 * [Option to disable custom notifications](#option-to-disable-custom-notifications)
+* [Not modificable headers in custom notifications](#not-modificable-headers-in-custom-notifications)
```

Comment:
Fixed in c4eb0130d595116bc4452aa6bcff3f3a49ad0960

Not only in .md. One of the new .test has been also changed (filename and test name)